pytri/__init__.py

```diff
@@ -169,28 +228,19 @@ def scatter(self, data, r=0.15, c=0x00babe):
             None
 
         """
-        d = data.tolist()
-        _js = ""
-        scatter_path, _ = split(__file__)
-        scatter_file = join(scatter_path, "js", "ScatterLayer.js")
-        with open(scatter_file, 'r') as fh:
-            _js += ";\n\n" + fh.read().strip()
-        _js += ("""
-        V['"""+self.uid+"""'].addLayer('scatter', new ScatterLayer({{
-            data: {},
-            radius: {},
-            colors: {}
-        }}))
-        """.format(
-            json.dumps(d),
-            r,
-            c
-        ))
-        display(Javascript(_js))
-        self.layers.add('scatter')
+        if isinstance(data, np.ndarray):
```

Comment:
I like this check, but do we want to do anything in particular if this doesn't pass? Are we assuming a list otherwise?

Reply:
Yeah, assuming a list of lists otherwise.... which is obv not quite how user-defined input works...